.claude/scripts/mcp_common.sh

```
@@ -0,0 +1,2094 @@
+    INSTALL_RESULTS["serena"]="ALREADY_EXISTS"
+    SUCCESSFUL_INSTALLS=$((SUCCESSFUL_INSTALLS + 1))
+else
+    # Remove existing serena server to reconfigure
+    ${MCP_CLI_BIN} mcp remove "serena" >/dev/null 2>&1 || true
+
+    # Add Serena MCP server using uvx with git repository
+    echo -e "${BLUE}  🔗 Adding Serena MCP server via uvx...${NC}"
+    log_with_timestamp "Attempting to add Serena MCP server via uvx"
+
+    # Use add-json for uvx configuration
+    local debug_env_var="MCP_${MCP_PRODUCT_NAME_UPPER}_DEBUG"
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)